static/bidder-info/vrtcal.yaml

```diff
@@ -1,10 +1,15 @@
 maintainer:
   email: "support@vrtcal.com"
+gvlVendorID: 706 
 capabilities:
   app:
     mediaTypes:
       - banner
       - video
+  site:
+    mediaTypes:
+      - banner
+      - video
```

Comment:
Please add "exemplary" json tests for site.

Reply:
> Please add "exemplary" json tests for site.

The exemplary SITE tests have been added. Thank you.